draft-ietf-httpbis-digest-headers.md

```diff
@@ -338,9 +338,9 @@ The Internet Assigned Numbers Authority (IANA) acts as a registry for
 digest-algorithm values.
 The registry contains the tokens listed below.
 
-Some algorithms, although registered, have since been found vulnerable:
-the "MD5" algorithm MUST NOT be used due to collision attacks [CMU-836068]
-and the "SHA" algorithm MUST NOT be used due
+Some digest-algorithms, although registered, rely on vulnerable algorithms:
+the "MD5" digest-algorithm MUST NOT be used due to collision attacks [CMU-836068]
+and the "SHA" digest-algorithm MUST NOT be used due
```

Comment:
Based on your other PR, I think this would be lower case md5 and sha?

Reply:
I split the PRs:

- #1256  lowercased only the *new* digest algorithms because that doesn't update anything;
- this will consistently use digest-algorithm
- the last PR will lowercase existing algorithms name.

Reply:
Makes sense, please proceed 🙂